ui/components/app/snaps/snap-ui-renderer/components/index.ts

```diff
@@ -46,4 +47,5 @@ export const COMPONENT_MAPPING = {
   Checkbox: checkbox,
   Tooltip: tooltip,
   Card: card,
+  Footer: footer,
```

Comment:
What about `Container`?

Reply:
`Container` is not rendered, I extract everything from it